client/src/App.js

```diff
@@ -1,12 +1,49 @@
+import Navbar from "./components/Navbar"
+import Aboutme from "./components/Aboutme"
+import Home from "./components/Home"
+import Work from "./components/Work"
+import Contact from "./components/Contact"
+import Project from "./components/Project"
+import Interest from "./components/Interest"
+import Footer from "./components/Footer"
 import React from 'react';
+
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hello portfolio!</h1>
-    </div>
-  );
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      page: "home",
+
+    }
+  }
+
+  contentHandler = (event) => {
+    this.setState({
+      page: event.target.id,
+    })
+  }
+
+  render() {
+
+    document.title= this.state.page
+
+    return (
+      <div>
+        <Navbar contentHandler={this.contentHandler} />
+        {this.state.page === "about" ? <Aboutme /> : null}
+        {this.state.page === "home" ? <Home /> : null}
+        {this.state.page === "work" ? <Work /> : null}
+        {this.state.page === "contact" ? <Contact /> : null}
+        {this.state.page === "projects" ? <Project /> : null}
+        {this.state.page === "interests" ? <Interest /> : null}
```

Comment:
The downside to doing navigation in this way is that you lose the use of your browser buttons, such as your back or forwards buttons. There are ways to fix this, one of them by implementing React Router Dom. This would allow you to have your history persist and give you controls of your browser buttons. 

If you wanted a run down on how to use we could schedule a time to go through that